src/kibana/components/vislib/lib/data.js

```diff
@@ -265,49 +230,37 @@ define(function (require) {
     };
 
     /**
-     * Calculates the largest y stack value among all data objects
      *
-     * @method getYStackMax
-     * @param series {Array} Array of data objects
-     * @returns {Number} Y stack max value
+     * @param series
+     * @param callback
+     * @returns {*}
      */
-    Data.prototype.getYStackMax = function (series) {
-      var isOrdered = (this.data.ordered && this.data.ordered.date);
-      var minDate = isOrdered ? this.data.ordered.min : undefined;
-      var maxDate = isOrdered ? this.data.ordered.max : undefined;
-
+    Data.prototype.getYExtents = function (series, calculation) {
       return d3.max(this.stackData(series), function (data) {
-        return d3.max(data, function (d) {
-          if (isOrdered) {
-            return (d.x >= minDate && d.x <= maxDate) ? d.y0 + d.y : undefined;
-          }
-
-          return d.y0 + d.y;
-        });
+        return d3.max(data, calculation);
       });
     };
 
     /**
+     * Calculates the largest y stack value among all data objects
+     *
+     * @method getYStackMax
+     * @param d {Object} data object
+     * @returns {Number} Y stack value
+     */
+    Data.prototype.getYStackMax = function (d) {
```

Comment:
Not sure what this method is supposed to be doing, but pretty sure the docs are out of date.